api/PayrollProcessor.Data.Persistence/Features/Employees/EmployeeRecord.cs

```diff
@@ -75,14 +75,11 @@ public static EmployeeRecord From(Employee employee)
                 };
             }
 
-            public static EmployeeRecord From(EmployeeNew employee)
-            {
-                var partitionId = Guid.NewGuid();
```

Comment:
Using this id caused later queries to fail since the generated id coming from the command source was not used.

Reply:
🐛 😉